### PR TITLE
Feature/home details

### PR DESCRIPTION
**Pull request by irene-haeryun-kwon**
1. Changed connection string so that it matches with the master branch
2. Added default statement for switch cases for Menu index and details file

### CODE REVIEW

File: PandaTea/appsettings.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "ConnectionStrings": {
     "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=aspnet-PandaTea-75335E9C-836C-45E0-B49A-E498C6BF442E;Trusted_Connection=True;MultipleActiveResultSets=true",
-    "PandaTeaConnection": "Data Source=.;Initial Catalog=pandateadb;Persist Security Info=True;User ID=sa;Password=Conestoga1"
+    "PandaTeaConnection": "Data Source=.;Initial Catalog=pandateadb;Persist Security Info=True;User ID=sa;Password=Irene123"
```
Comment (JadenAhn) [inline PandaTea/appsettings.json]: Can you reset the password to the original one?

File: PandaTea/Views/Menu/Details.cshtml [hunk 2]
```diff
@@ -1,43 +1,234 @@
...
+                        break;
+                    case 4:
+                        <div>Coffee Chill</div>
+                        break;
+                    case 5:
+                        <div>Zen Matcha</div>
+                        break;
+                    case 6:
+                        <div>Taro Basic</div>
+                        break;
```
Comment (JadenAhn) [inline PandaTea/Views/Menu/Details.cshtml]: Can you add a default case for the code quality? Same as below Case statements. :)

File: PandaTea/appsettings.json (hunk 1 above)
Comment (irene-haeryun-kwon) [reply PandaTea/appsettings.json]: Sure

File: PandaTea/Views/Menu/Details.cshtml (hunk 2 above)
Comment (irene-haeryun-kwon) [reply PandaTea/Views/Menu/Details.cshtml]: Good catch

**Pull request merged**